### DISCUSSION TITLE
500 error on inputing custom name for site

### DISCUSSION THREAD

**Discussion by dentedpixel** [community/community#40153] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I have been trying to connect my own domain (dentedpixel.com) to the page https://dentedpixel.github.io/ for many days now, but every time I try and enter this domain I get a 500 error.

It doesn't seem to matter what domain I give it, any entry results in a 500 error.

**Comment by radar-blip** [comment]
Seems to be working now 🖍️
